### PR TITLE
Assignment 2 files

### PR DESCRIPTION
**Pull request by mcaminome**

### CODE REVIEW

File: Assignment2/mcaminome-MacarenaCamino.ttl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+@prefix user: <http://www.ontologies.org/ontologies/user#>
+@prefix sensor: <http://www.ontologies.org/ontologies/sensor#>
+@prefix measurement: <http://www.ontologies.org/ontologies/measurement#>
+
+:Class01	class:includes:Sensor029;
+			class:includes:Computer101.
+							
+	:Sensor029	sensor:hasMeasurement:Measurement8401.
+
+		:Measurement8401	measurement:hasTemperature:"29"^^xsd:integer;
```
Comment (ocorcho) [inline Assignment2/mcaminome-MacarenaCamino.ttl:13]: Have you tried with a validator? You should always try to leave a blank space (at least) between elements in the triple

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Review by mcaminome — COMMENTED** [review]
Update Assignment 2 and upload assignment 2b

**Comment by ocorcho** [comment]
Assignments 1 and 2 are now complete. 
Now @paoespinozarias will check assignment 2b

File: Assignment2/mcaminome-fromJSONtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{
+	"@context": {
+  	"people": "http://www.oeg-upm.net/ontologies/people",
```
Comment (paoespinozarias) [inline Assignment2/mcaminome-fromJSONtoJSON-LD.jsonld:3]: Add / or # at the end of your URI definition

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

File: Assignment2/mcaminome-fromJSONtoJSON-LD.jsonld [hunk 3]
```diff
@@ -0,0 +1,62 @@
+{
+	"@context": {
+  	"people": "http://www.oeg-upm.net/ontologies/people",
+    "fullname": "people:fullname",
+    "hometown": "people:hometown",
+    "age": "people:age",
+    "almamater":"people:almamater",
+    "parents":"people:parents",
+    "mother":"parents:mother",
+    "father":"parents:father",
```
Comment (paoespinozarias) [inline Assignment2/mcaminome-fromJSONtoJSON-LD.jsonld:10]: These attribute definitions work, but semantically they aren't ok. The best way is to use a prefix and add the name of the concept/property you want to use e.g. "mother":"yourvocab:mother".
The same comment applies to the combination you did in the subject attributes definition. 
You don't need to change them, but remember to provide good definitions for the future.

File: Assignment2/mcaminome-fromRDFtoJSON-LD.jsonld [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    {
+      "@id": "user:User10A",
+      "properties:hasName": {
+        "@value": "Pedro",
+        "@type": "xsd:string"
+      }
+    }
+  ]
+}
```
Comment (paoespinozarias) [inline Assignment2/mcaminome-fromRDFtoJSON-LD.jsonld:52]: This file is ok.

**Comment by ocorcho** [comment]
I am surprised that you have not taken into account the latest set of comments provided by @paoespinozarias 
Given this situation, I have decided to close this pull request and not accept anything that has been submitted. Contact us directly if you wish your work to be considered.

**Pull request closed without merging**